french_law/js/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "french_law",
-  "version": "0.5.0",
+  "name": "@catala-lang/french-law",
+  "version": "0.5.2",
```

Comment:
Same bump to 0.8.0

Reply:
The frequency of updates may go faster for the french law, no?

Reply:
Yes, but let's keep the major (and possibly minor) versionning in sync with the version of Catala.